integrated_channels/cornerstone/views.py

```diff
@@ -32,96 +37,6 @@ class BaseViewSet(generics.ListAPIView):
     renderer_classes = [renderers.JSONRenderer]
 
 
-class CornerstoneCoursesUpdates(BaseViewSet):
```

Comment:
originally, we were going to use a new endpoint for support of the `If-Modified-Since` header. it was never used and I plan to merge that functionality into the existing endpoint instead. removing this code for now.